actions/st2_prep_release_for_st2_pkg.sh

```diff
@@ -101,8 +101,9 @@ fi
 # Update the st2 version at circle.yml
 CIRCLE_YML_FILE="circle.yml"
 echo "Setting version in ${CIRCLE_YML_FILE} to ${BRANCH}..."
-sed -i -e "s/\(ST2_GITREV:[ ]*\).*/\1${BRANCH}/" ${CIRCLE_YML_FILE}
-sed -i -e "s/\(ST2MISTRAL_GITREV:[ ]*\).*/\1${MISTRAL_VERSION}/" ${CIRCLE_YML_FILE}
+# capture group one is to ensure proper indentation. Capture group two is actual var definition
+sed -i -e "s/\(.*\)# \(ST2_GITREV:[ ]*\).*/\1\2${BRANCH}/" ${CIRCLE_YML_FILE}
```

Comment:
IMO you made this regex too complex (and I love me some regex)

`sed -e "s/# ST2_GITREV: \"\"/ST2_GITREV: ${BRANCH}/" %{CIRCLE_YML_FILE}`

If you wanted to make this more robust (which isn't needed) you should use `\s*` or `\s\+` (Thanks `sed` for all the escapes) to match white space. Matching everything `\(.*\)` is undesirable since it will allow syntax errors when you don't want them without failing.